### PR TITLE
[AI4EUEXP-40] Delete model not possible for dockerized-uri models

### PR DESCRIPTION
**Pull request by HelgeS**
This pull request adds an additional check whether a docker image (artifact type DI) can actually be deleted by Acumos.

If the image is not in the own registry, no deletion attempt is made.

TODO
- [X] Test deletion of models with external images
- [ ] Test deletion of models with images in the own registry

### CODE REVIEW

File: acumos-portal-fe/pom.xml [hunk 1]
```diff
@@ -23,15 +23,15 @@ limitations under the License.
...
+	<version>4.0.14-ai4eu-v4</version>
 	<packaging>war</packaging>
 	<name>Acumos Portal frontend server</name>
 	<description>Acumos Portal AngularJS Front End</description>
 
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.16.RELEASE</version>
+		<version>2.1.1.RELEASE</version>
```
Comment (peschue) [inline acumos-portal-fe/pom.xml:34]: why is it necessary to change to a newer spring boot framework?

File: pom.xml [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 	<artifactId>portal-marketplace-parent</artifactId>
-	<version>4.0.14-ai4eu-v3</version>
+	<version>4.0.14-ai4eu-v4</version>
 	<packaging>pom</packaging>
 	<name>Acumos Portal Marketplace parent project</name>
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>1.8</java.version>
+		<java.version>11</java.version>
```
Comment (peschue) [inline pom.xml:13]: are we sure this will work well for all components?

File: acumos-portal-fe/pom.xml (hunk 1 above)
Comment (HelgeS) [reply acumos-portal-fe/pom.xml:34]: I mixed another pull request with this one. The updated spring and java is already in master.
It was partially updated by upstream, but the way they did it broke the ability to run it locally.
The updated Spring & Java has been running on acumos-dev until the environment was upgraded and no new issues were discovered (to my knowledge).

If it breaks, we can still revert it.

Comment (peschue) [reply acumos-portal-fe/pom.xml:34]: Ah, then that's great!

**Comment by HelgeS** [comment]
Merging this since it works for models on-boarded with dockerized model uri and fixes the main issue we have.

**Pull request merged**